superset/assets/javascripts/explore/components/PairedTTestTableContainer.jsx

```diff
@@ -0,0 +1,161 @@
+/* eslint camelcase: 0 */
+import React from 'react';
+import PropTypes from 'prop-types';
+import dist from 'distributions';
+
+import { connect } from 'react-redux';
+import { Table } from 'react-bootstrap';
```

Comment:
We've been using `reactable`, it offers good features out of the box (sorting, filtering)